release_build.sh

```diff
@@ -25,24 +25,35 @@ fi
 KBS_VERS_ARRAY=(${KBS_VERS})
 printf "%s\n" "${KBS_VERS_ARRAY[@]}"
 
-echo "Fetching list of latest CRI-O patch releases"
-echo "================================================="
-
-git ls-remote --tags --sort=-v:refname https://github.com/cri-o/cri-o \
-    | grep -v "{}" \
-    | awk '{ print $2}' \
-    | cut --delimiter='/' --fields=3 \
-    > crio.txt
-
-CRIO=()
-for r in "${KBS_VERS_ARRAY[@]}"; do
-    if ! grep -q "v${r%.*}" crio.txt; then
-        echo "Skipping $r"
-        continue
-    fi
-    version=$(cat crio.txt | grep "v${r%.*}" | head -n1)
-    CRIO+=( "crio-${version:1}" )
-done
+# fetch_releases returns available for a given software
+# based on Kubernetes major versions.
+function fetch_releases {
+	local software="${1}"
+	local file; file=$(mktemp)
```

Comment:
Forgot to point that out earlier, but we probably want to do `rm -f "${file}"` at the end of the function, no?